src/Symfony/Component/Testing/composer.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "symfony/testing",
+    "type": "library",
+    "description": "Symfony Testing Component",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
+            "email": "p@tchwork.com"
+        }
+    ],
+    "require": {
+        "php": ">=5.3.3"
+    },
+    "autoload": {
+        "psr-0": { "Symfony\\Component\\Testing\\": "" },
+        "files": [ "bootstrap.php" ]
+    },
+    "target-dir": "Symfony/Component/Testing",
```

Comment:
please use PSR-4 instead of PSR-0 + target-dir. target-dir is a deprecated feature in Composer